src/main/menu.ts

```diff
@@ -135,13 +135,18 @@ export function buildMenu(windowManager: WindowManager) {
           }
         }
       ]),
+
       { type: "separator" },
-      {
-        role: "close",
-        label: "Close Window"
-      },
+
+      ...(isMac ? [
```

Comment:
We don't need "Close window" command on win. Otherwise, it will just duplicate the `Exit` one.